### ISSUE TITLE
Include the motion estimates visualization report in the subject.html report file

### ISSUE THREAD

**Issue by jandrushko** [poldracklab/fmriprep#1894]
Would it be possible to include the 6 DoF rigid body motion estimates for rotations and translations by MCFLIRT in the .html report? In FSL it provides a visual of the estimates across time in addition to providing the means of total motion and I find this very handy to quickly check for total amount of motion in addition to see if there is any task correlated motion in the data across the run (task-based fmri)

**Comment by oesteban** [comment]
Wouldn't the FD exactly address this?

**Comment by jandrushko** [comment]
I suppose it will. I just found that having the 6 motion parameters separate (translations and rotations) like is in FSL is easier to uncover potentially task correlated motion. For example, when I ran the fmriprep pipeline and a normal FSL workflow on the same data, the FSL motion estimates clearly showed some task correlated movement in one or multiple directions of motion, whereas this was not as evident in the FD plot from fmriprep. Just from a visual standpoint I found it easier to quickly assess the data for the 6 motion estimates plotted across the volumes. If the vast majority don't agree with this added visual inspection then consider this issue closed. Thanks

**Comment by effigies** [comment]
Yeah, FD is good for identifying motion spikes, but does not capture drifts in motion parameters.

I think it would be reasonable to include them, but that section is starting to get quite cluttered. It would be worth thinking about how to better present, especially if we're proposing adding 6 more time series.

Another thing we could consider is creating simple boxcar functions from `events.tsv` and showing any regressors that correlate above some threshold, similar to how we show correlations with the global signal.

**Comment by acjanes** [comment]
I would also like to see motion information in the output. I would find this helpful in inspecting the AROMA output in regards to ensuring that ones data isn't over-contaminated by motion and thus removing meaningful signals as well.
